tagsets/orsee_mysql.php

```diff
@@ -247,5 +247,39 @@ function dump_array($array,$title="",$dolang=true) {
     echo '</TABLE>';
 }
 
+function version_to_integer($version) {
+    $varray=explode(".",$version);
+    $version_int=$varray[0]*10000+$varray[1]*100+$varray[2]*1;
```

Comment:
Also, this is important, perhaps before we are not too far, I suggest to use timestamp as 'database_version' (e.g. 2017112700) There might be more than one DB upgrade between releases (e.g. multiple pull requests with DB changes), thus each pull request brings own changes with timestamp version and there will be no need to merge all of them in a single 'if' statement within or_upgrade_database prior to release. This version_to_integer function will not be required at all in this case.